### ISSUE TITLE
openssl-1.1.0c: build errors on sid/stretch

### ISSUE THREAD

**Issue by linuxmaniac** [kamailio/kamailio#958]
```
apt-cache policy openssl
openssl:
  Installed: 1.1.0c-2
  Candidate: 1.1.0c-2
  Version table:
 *** 1.1.0c-2 100
        100 /var/lib/dpkg/status
```

```
make every-module group_include="ktls"
make[1]: Entering directory '/code'
make -C src/ every-module 
make[2]: Entering directory '/code/src'
CC (gcc) [M crypto.so]		crypto_uuid.o
crypto_uuid.c:36:26: fatal error: openssl/rand.h: No such file or directory
 #include <openssl/rand.h>
                          ^
compilation terminated.
make[3]: *** [crypto_uuid.o] Error 1
../../Makefile.rules:100: recipe for target 'crypto_uuid.o' failed
```

**Comment by linuxmaniac** [comment]
this is trunk but this has been reported for 4.4 at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=852905

```

During a rebuild of all packages in sid, your package failed to build on
amd64.

Relevant part (hopefully):
> gcc -fPIC -DPIC -funroll-loops -Wcast-align -m64 -minline-all-stringops -falign-loops -ftree-vectorize -fno-strict-overflow -Wall -Wdate-time -D_FORTIFY_SOURCE=2 -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -DVERSION_NODATE   -DNAME='"kamailio"' -DVERSION='"4.4.4"' -DARCH='"x86_64"' -DOS='linux_' -DOS_QUOTED='"linux"' -DCOMPILER='"gcc 6.3.0"' -D__CPU_x86_64 -D__OS_linux -DSER_VER=4004004 -DCFG_DIR='"/etc/kamailio/"' -DRUN_DIR='"/var/run/kamailio/"' -DPKG_MALLOC -DSHM_MEM -DSHM_MMAP -DDNS_IP_HACK -DUSE_MCAST -DUSE_TCP -DDISABLE_NAGLE -DHAVE_RESOLV_RES -DUSE_DNS_CACHE -DUSE_DNS_FAILOVER -DUSE_DST_BLACKLIST -DUSE_NAPTR -DWITH_XAVP -DMEM_JOIN_FREE -DF_MALLOC -DQ_MALLOC -DTLSF_MALLOC -DDBG_SR_MEMORY -DUSE_TLS -DTLS_HOOKS -DUSE_CORE_STATS -DSTATISTICS -DMALLOC_STATS -DWITH_AS_SUPPORT -DUSE_SCTP -DFAST_LOCK -DADAPTIVE_WAIT -DADAPTIVE_WAIT_LOOPS=1024 -DCC_GCC_LIKE_ASM -DHAVE_GETHOSTBYNAME2 -DHAVE_UNION_SEMUN -DHAVE_SCHED_YIELD -DHAVE_MSG_NOSIGNAL -DHAVE_MSGHDR_MSG_CONTROL -DHAVE_ALLOCA_H -DHAVE_TIMEGM -DHAVE_SCHED_SETSCHEDULER -DHAVE_EPOLL -DHAVE_SIGIO_RT -DSIGINFO64_WORKARROUND -DUSE_FUTEX -DHAVE_SELECT  -I/usr/include/x86_64-linux-gnu -I/usr/local/ssl/include -DSER_MOD_INTERFACE -DMOD_NAME='"auth_identity"' -c auth_http.c -o auth_http.o -MMD -MP
> auth_http.c:35:25: fatal error: openssl/pem.h: No such file or directory
>  #include <openssl/pem.h>
>                          ^
> compilation terminated.
> ../../Makefile.rules:97: recipe for target 'auth_http.o' failed
> make[2]: *** [auth_http.o] Error 1

The full build log is available from:
   http://aws-logs.debian.net/2017/01/28/kamailio_4.4.4-1_unstable.log
```

**Comment by linuxmaniac** [comment]
checking build dependencies

**Comment by miconda** [comment]
Is libssl installed from source? I see /usr/local/ssl/include in the include path for gcc.

Can you list the content of that folder? Maybe they changed the path and they don't have anymore the pattern like <openssl/*.h>

**Comment by linuxmaniac** [comment]
seems like libssl-dev now is missing from the list of dependencies, fixing.

**Comment by linuxmaniac** [comment]
fixed by c293df833
